client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/SslConfigurationBuilder.java

```diff
@@ -256,8 +259,10 @@ public ConfigurationBuilder withProperties(Properties properties) {
       if (typed.containsKey(ConfigurationProperties.TRUST_STORE_FILE_NAME))
          this.trustStoreFileName(typed.getProperty(ConfigurationProperties.TRUST_STORE_FILE_NAME, trustStoreFileName, true));
 
-      if (typed.containsKey(ConfigurationProperties.TRUST_STORE_PATH))
+      if (typed.containsKey(ConfigurationProperties.TRUST_STORE_PATH)) {
+         HOTROD.deprecatedConfigurationProperty(ConfigurationProperties.TRUST_STORE_PATH);
```

Comment:
I would move the warning to `trustStorePath()`. Configuring the client programmatically doesn't show any warning.